hub/src/server/http.js

```diff
@@ -107,6 +109,45 @@ export function makeHttpServer(config: MakeHttpServerConfig & HubServerConfig) :
         })
   })
 
+  app.post(
+    /^\/revoke-all\/([a-zA-Z0-9]+)\/?/, 
+    express.json(),
+    (req: express.request, res: express.response) => {
+      // sanity check...
+      if (req.headers['content-length'] > 4096) {
+        writeResponse(res, { mesasge: 'Invalid JSON: too long'}, 400)
+        return
+      }
+
+      if (!req.body || !req.body.oldestValidTimestamp) {
+        writeResponse(res, { mesasge: 'Invalid JSON: missing oldestValidTimestamp'}, 400)
+        return
+      }
+
+      const address = req.params[0]
+      const oldestValidTimestamp: number = parseInt(req.body.oldestValidTimestamp)
```

Comment:
This needs to be surrounded with try/catch in case the caller doesn't send a number here.

Reply:
Lol, once again I set my expectations too high for JS.  Thanks for clarifying :+1: